docs/Local_Development_Tutorial.md

```diff
@@ -0,0 +1,168 @@
+```sh
+kubectl port-forward marklogic-0 8001 8000 7997
+```
+ If you want to forward other ports, just append them to the command separated by a space.  
+For example: 
+
+```sh
+kubectl port-forward marklogic-0 8001 8000 7997 7996 7888 1234 1337
+```
+
+
+- To complete this step, access your browser and navigate to `localhost:8001`. You should see the MarkLogic Server Admin interface.
+If you are unable to see the MarkLogic Server Admin interface, see the [Debugging](#Debugging) section to gather more information about the cluster and potential errors. 
+
+- See the [Cleanup](#Cleanup) section in order to teardown the cluster when you are finished. 
+
+# Debugging
+This Debugging section contains useful commands to help debug a Kubernetes cluster running MarkLogic Server. Additional information and commands can be found here: https://kubernetes.io/docs/tasks/debug-application-cluster/debug-running-pod/
+
+This command provides additional information about the state of each pod in the cluster. 
+
+```sh
+kubectl describe pods
+```
+The command outputs a large amount of data and contains a lot of information. Pay attention to the events section near the bottom of the output to view the state of events, and what occurred during the startup of the pod.  
+For Example:
+
+```
+...
+Events:
+  Type     Reason     Age   From               Message
+  ----     ------     ----  ----               -------
+  Normal   Scheduled  13m   default-scheduler  Successfully assigned default/marklogic-0 to minikube
+  Normal   Pulled     13m   kubelet            Container image "store/marklogicdb/marklogic-server:10.0-8.3-centos-1.0.0-ea3" already present on machine
+  Normal   Created    13m   kubelet            Created container marklogic
+  Normal   Started    13m   kubelet            Started container marklogic
+  Warning  Unhealthy  13m   kubelet            Startup probe failed: ls: cannot access /var/opt/MarkLogic/ready: No such file or directory
+```
+**Note:** It is OK that the startup probe failed once. The probe will poll a few times. 
+
+-----
+
+Run the following command to see the logs for a specific pod:
+```sh
+kubectl logs {POD_NAME} 
+```
+The `{POD_NAME}` can be found with the `kubectl get pods` command. 
+
+Example output: 
+
+```
+2022-03-28 16:53:00.127 Info: Memory 4% phys=5812 size=536(9%) rss=275(4%) anon=205(3%) file=9(0%) forest=853(14%) cache=1920(33%) registry=1(0%)
+2022-03-28 16:54:01.630 Info: Merging 1 MB from /var/opt/MarkLogic/Forests/Security/00000002 to /var/opt/MarkLogic/Forests/Security/00000003, timestamp=16484858400268930
+2022-03-28 16:54:01.657 Info: Merged 1 MB at 37 MB/sec to /var/opt/MarkLogic/Forests/Security/00000003
+2022-03-28 16:54:04.155 Info: Deleted 2 MB at 1314 MB/sec /var/opt/MarkLogic/Forests/Security/00000002
+```
+
+-----
+
+Run the following command to see the logs for a specific pod : 
+
+```sh
+kubectl exec --stdin --tty {POD_NAME} -- /bin/bash
```

Comment:
This looks good as well. Just wondering if we can use -it to replace --stdin --tty for a more concise command.

Reply:
Good call, I tried this when testing and probably garbled it somehow since it didn't work for me then: 0c45940 Updated